cookbooks/aws-parallelcluster-platform/spec/unit/recipes/enable_services_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe 'aws-parallelcluster-platform::enable_services' do
+  for_all_oses do |platform, version|
```

Comment:
Isn't the enable:Services recipe meant for AL2023 only?

Reply:
Removed after refactoring the code